Added readme and force gaze option (#81)
ami-iit/yarp-device-openxrheadset: src/devices/openxrheadset/README.md

| `left_image_rotation` | double | `0.0` | Counter-clockwise rotation of the left eye image, in radians. |
| `right_azimuth_offset` | double | `0.0` | Azimuth offset for the right eye image, in radians. |
| `right_elevation_offset` | double | `0.0` | Elevation offset for the right eye image, in radians. |
| `right_image_rotation` | double | `0.0` | Counter-clockwise rotation of the right eye image, in radians. |
| `eye_z_position` | double | `-1.0` | The Z position of the eye quad layers. The value is forced to be negative and at least `0.01` in absolute value. |

---

## OpenXR Interface Settings

| Parameter | Type | Default | Description |
|-----------|------|---------|-------------|
| `vr_poses_prediction_in_ms` | double | `0.0` | The prediction time for pose tracking, in milliseconds. |
| `hide_window` | bool (flag) | `true` if `use_native_quad_layers` is set, `false` otherwise | If set (or set to `true`), hides the desktop mirror window. |
| `render_in_play_space` | bool (flag) | `false` | If set (or set to `true`), renders in the play space instead of the view reference space. |
| `no_gaze` | bool (flag) | `false` | If set (or set to `true`), disables gaze tracking. |

> | `force_use_gaze` | bool (flag) | `false` | If set (or set to `true`), forces the use of gaze tracking even if the runtime reports that the system does not support eye gaze interaction. Has no effect if `no_gaze` is set. |

Reviewer: ```suggestion
| `force_use_gaze` | bool (flag) | `false` | If set (or set to `true`), forces the use of gaze tracking even if the runtime reports that the system does not support eye gaze interaction. Device won't start if both `no_gaze` and this flag are set. |
```
